src/Http/Http.Results/src/PhyiscalFileResult.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Http.Result
+{
+    /// <summary>
+    /// A <see cref="PhysicalFileResult"/> on execution will write a file from disk to the response
+    /// using mechanisms provided by the host.
+    /// </summary>
+    internal sealed partial class PhysicalFileResult : FileResult, IResult
```

Comment:
Not quite the right line for this comment, but the file name has a typo: `PhyiscalFileResult.cs` => `PhysicalFileResult.cs`